dali/operators/image/crop/crop_attr.cc

```diff
@@ -200,7 +231,8 @@ TensorShape<> CropAttr::CalculateAnchor(const span<float>& anchor_norm,
     DALI_ENFORCE(anchor_norm[dim] >= 0.0f && anchor_norm[dim] <= 1.0f,
                  "Anchor for dimension " + std::to_string(dim) + " (" +
                      std::to_string(anchor_norm[dim]) + ") is out of range [0.0, 1.0]");
-    anchor[dim] = std::roundf(anchor_norm[dim] * (input_shape[dim] - crop_shape[dim]));
+    float anchor_f = anchor_norm[dim] * (input_shape[dim] - crop_shape[dim]);
+    anchor[dim] = round_fn_(anchor_f);
```

Comment:
While we're at it, I'd bump precision to double. It's not very likely to happen, but we might run into some bizarre 1D case that would be more than 8M pixels wide.